frontend/apps/crates/components/src/profile_image/config.rs

```diff
@@ -0,0 +1,20 @@
+use shared::domain::{image::ImageId};
+
+pub const STR_WRONG_FILE_TYPE: &str = "Oh no! We don't accept this type of file.";
+pub const STR_FILE_TOO_LARGE: &str = "Oh no! This file is too heavy. Maximum file size: ";
+
+pub struct ProfileImageConfig {
+    pub close: Box<dyn Fn()>,
```

Comment:
Do we need the close callback? Don't think it's ever called

Reply:
Agreed!